docs/manuals/deployment/docker/production.md

```diff
@@ -0,0 +1,79 @@
+## Prerequisites
+
+- Knowledge of Docker.
+- [Prepare your application for deployment](../prepare-taipy-for-deployment.md)
+- [Followed steps for development](development.md)
+
+
+## Production ready Dockerfile
+
+The following Dockerfile contains minimum configuration for put your application in production.
```

Comment:
"contains minimum configuration for put your application in production"
->
"contains the minimum configuration settings for you to deploy your application to production"